tests/unit/cube/qb/test_qb_unit.py

```diff
@@ -19,35 +19,6 @@
 SCHEMA_PATH_FILE = APP_ROOT_DIR_PATH / "schema" / "cube-config" / "v1_0" / "schema.json"
 
 
-def test_new_unit_base_unit_validation():
```

Comment:
Have these tests been replaced?

Reply:
https://github.com/GSS-Cogs/csvcubed/blob/9b259fcf2f2b4468c0e90c0ed528b2b81c186487/tests/unit/utils/test_validations.py#L763-L805

I think they were included in `test_validations.py`